src/groundStation/uTransceiver.py

```diff
@@ -0,0 +1,133 @@
+import ctypes
+from ctypes import cdll, c_int, c_char
+import sys
+import zmq
+import threading
+import os.path
+import socket
+import time
+
+
+
+# command format follows https://docs.google.com/spreadsheets/d/1zNhxhs0KJCp1187Vm3-zAzQHCY31f77l-0nlQmfXu1w/edit#gid=565953736
```

Comment:
Access denied to this. Relevant information should be placed either in a docstring here or a documentation file

Reply:
Would be nice to be made available so I can plan on how the website should handle these commands